checkov/arm/checks/resource/AzureDefenderOnStorage.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Any, Dict
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.arm.base_resource_check import BaseResourceCheck
+
+
+class AzureDefenderOnStorage(BaseResourceCheck):
+    def __init__(self) -> None:
+        name = "Ensure that Azure Defender is set to On for Storage"
+        id = "CKV_AZURE_84"
+        supported_resources = ("Microsoft.Security/pricings",)
+        categories = (CheckCategories.GENERAL_SECURITY,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf: Dict[str, Any]) -> CheckResult:
+        properties: Dict[str, Any] = conf.get("properties", {})
+        pricingTier = properties.get("pricingTier", "")
+        resourceType = properties.get("resourceType", "")
+        return (
+            CheckResult.PASSED
+            if resourceType != "Microsoft.Security/pricings" or pricingTier == "Standard"
```

Comment:
You can remove the `if resourceType != "Microsoft.Security/pricings"` - in ARM it will always be this type.
(Just for terraform it can be other types as well)